lsp-test/src/Language/LSP/Test.hs

```diff
@@ -605,6 +622,22 @@ executeCodeAction action = do
           let req = TRequestMessage "" (IdInt 0) SMethod_WorkspaceApplyEdit (ApplyWorkspaceEditParams Nothing e)
             in updateState (FromServerMess SMethod_WorkspaceApplyEdit req)
 
+-- |Resolves the provided code action.
+resolveCodeAction :: CodeAction -> Session CodeAction
+resolveCodeAction ca = do
+  rsp <- request SMethod_CodeActionResolve ca
+  case rsp ^. L.result of
+    Right ca -> return ca
+    Left error -> throw (UnexpectedResponseError (SomeLspId $ fromJust $ rsp ^. L.id) error)
```

Comment:
```suggestion
    Left er -> throw (UnexpectedResponseError (SomeLspId $ fromJust $ rsp ^. L.id) er)
```
just to avoid the unnecessary shadowing of `Prelude.error`